### ISSUE TITLE
createMetaData error?

### ISSUE THREAD

**Issue by shiffman** [PLhery/node-twitter-api-v2#93]
Hello! I'm so excited to find this package! I've been using Twit and other node.js implementations for many years and have been hoping to find something up-to-date and actively maintained!

I ran across an issue attempting to add "alt text" to an image, this fails for me with a `400` error:

```javascript
const mediaId = await client.v1.uploadMedia('rainbow.png', { type: 'png' });
await client.v1.createMediaMetadata(`${mediaId}`, { alt_text: 'Rainbow!' });
```

However, I am able to get it to work by calling `post()` directly.

```javascript
const mediaId = await client.v1.uploadMedia('rainbow.png', { type: 'png' });
await client.v1.post('media/metadata/create.json', {
  media_id: `${mediaId}`,
  alt_text: {
    text: 'Rainbow!',
  },
});
```

My repo of examples is in progress here: https://github.com/shiffman/Twitter-Bots-F21

**Comment by alkihis** [comment]
Hi,
1.6.0 version, that fixes the issue, is now available on npm :)
You can safely use `.createMediaMetadata` again!

Thanks you for the report!
